www-apps/gitea/gitea-1.19.0.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 2016-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fcaps go-module tmpfiles systemd flag-o-matic
+
+DESCRIPTION="A painless self-hosted Git service"
+HOMEPAGE="https://gitea.io https://github.com/go-gitea/gitea"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/go-gitea/gitea.git"
+else
+	SRC_URI="https://github.com/go-gitea/gitea/releases/download/v${PV}/gitea-src-${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86"
+fi
+
+S="${WORKDIR}/${PN}-src-${PV}"
+
+LICENSE="Apache-2.0 BSD BSD-2 ISC MIT MPL-2.0"
+SLOT="0"
+IUSE="+acct pam sqlite pie"
+
+DEPEND="
+	acct? (
+		acct-group/git
+		acct-user/git[gitea] )
+	pam? ( sys-libs/pam )"
+RDEPEND="${DEPEND}
+	dev-vcs/git"
+
+DOCS=(
+	custom/conf/app.example.ini CONTRIBUTING.md README.md
+)
+FILECAPS=(
+	-m 711 cap_net_bind_service+ep usr/bin/gitea
+)
+
+RESTRICT="test"
+
+src_prepare() {
+	default
+
+	local sedcmds=(
+		-e "s#^ROOT =#ROOT = ${EPREFIX}/var/lib/gitea/gitea-repositories#"
+		-e "s#^ROOT_PATH =#ROOT_PATH = ${EPREFIX}/var/log/gitea#"
+		-e "s#^APP_DATA_PATH = data#APP_DATA_PATH = ${EPREFIX}/var/lib/gitea/data#"
+		-e "s#^HTTP_ADDR = 0.0.0.0#HTTP_ADDR = 127.0.0.1#"
+		-e "s#^MODE = console#MODE = file#"
+		-e "s#^LEVEL = Trace#LEVEL = Info#"
+		-e "s#^LOG_SQL = true#LOG_SQL = false#"
+		-e "s#^DISABLE_ROUTER_LOG = false#DISABLE_ROUTER_LOG = true#"
```

Comment:
Could we start cleaning these up? 

```
 * SED: some expressions in the following did not cause any changes
 *     sed -i "${sedcmds[@]}" custom/conf/app.example.ini || die;
 * no-op: -e s#^ROOT =#ROOT = /var/lib/gitea/gitea-repositories#
 * no-op: -e s#^ROOT_PATH =#ROOT_PATH = /var/log/gitea#
 * no-op: -e s#^APP_DATA_PATH = data#APP_DATA_PATH = /var/lib/gitea/data#
 * no-op: -e s#^HTTP_ADDR = 0.0.0.0#HTTP_ADDR = 127.0.0.1#
 * no-op: -e s#^LEVEL = Trace#LEVEL = Info#
 * no-op: -e s#^LOG_SQL = true#LOG_SQL = false#
 * no-op: -e s#^DISABLE_ROUTER_LOG = false#DISABLE_ROUTER_LOG = true#
```
The `${EPREFIX}` ones are probably still useful, but there seem to be many outdated sed calls here.